doc/tests/verification_plan.rst

```diff
@@ -0,0 +1,155 @@
+.. _Verification_plan:
+
+Zephyr Verification Plan
+########################
+
+This document provides a general introduction of the Zephyr Verification
+Plan. Zephyr provides test suite for various functionalities offered in
+the project.
+
+**Kernel services and capabilities**
+    Services offered by kernel like memory management, scheduling,
+    interrupt services, timers and data passing mechanisms are verified
+    with API level testing and functionality testing.
+
+**Networking services**
+    Various protocols including HTTP, COAP, MQTT, ARP, 6lo, TCP, UDP
+    and other protocols are verified.
+
+**Sensor subsystem support**
+    Zephyr sensor driver is validated to measure various physical
+    parameters such as temperature, pressure, ambience, humidity,
+    light intensity, proximity and other real time sensory information.
+
+**File system**
+    Newtron Flash Filesystem (NFFS) and FATFS support, FCB
+    (Flash Circular Buffer) for memory constrained applications, and
+    file system enhancements for logging and configuration are verified.
+
+**Device Driver support**
+    Device drivers such as GPIO, I2C, I2S, SPI, RTC, UART, AIO and
+    various other driver functionalities are verified.
+
+**Power management services**
+    Power management services such as tickless idle and an advanced
+    idling infrastructure are also verified.
+
+**POSIX compliance**
+    APIs introduced to make Zephyr POSIX complaint are verified
+    through API level testing.
+
+Verification Strategy and Approach
+**********************************
+
+Zephyr test suite contains various test types such as
+
+Component level test
+
```

Comment:
For these dictionary lists, delete the blank line here (as you did in the list above).  You don't need to put the bold tags on the terms (that's done for you by the style), so change this to:

    Component level test
        Test against a single Zephyr module.

    API level test
        Test correctness of Zephyr APIs made available for application
        developer.

etc.